src/components/pages/Home/Notifications.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import { Badge, Dropdown, List, Menu } from 'antd';
+import { BellOutlined } from '@ant-design/icons';
+
+const NotificationListItem = props => {
+  const { notification, onClickView, onClickDismiss } = props;
+
+  const handleClick = key => {
```

Comment:
Looks like you'll need to pass in the event here as well.
`const handleClick = (evt, key) => {`